### ISSUE TITLE
Grasp plugin doesn't work with a gazebo issue

### ISSUE THREAD

**Issue by saikishor** [JenniferBuehler/gazebo-pkgs#25]
There is a gazebo issue that makes the gripper open and close due to an unknown reason, while you do a motion grasping an object. I thought this plugin would fix that, but as the gripper opens and closes, the object is falling down. Do you know any workaround?.

Thank you,

**Comment by JenniferBuehler** [comment]
This plugin is meant to release the object once the gripper is not touching it any more. So it's not suitable to fix this bug. Is there an issue in Gazebo for this bug?

**Comment by saikishor** [comment]
@JenniferBuehler  No I don't think there is such issue on this bug. I don't know why it is happening, but it is totally strange as the same thing worked in the Gazebo 2.2.3, but now when upgraded to kinetic we use gazebo 7.12.0 here comes the problem of the gripper, when doing a motion after grasping the object. I created a question in the gazebo answers(http://answers.gazebosim.org/question/20490/issue-with-the-post-grasp-motion-in-gazebo/). I recently created an issue on gazebo(https://bitbucket.org/osrf/gazebo/issues/2528/issue-with-the-post-grasp-motion-with-the)
